cmd/applications-rp/radius-dev.yaml

```diff
@@ -2,7 +2,7 @@
 environment:
   name: Dev
   roleLocation: "global"
-storageProvider:
+databaseProvider:
```

Comment:
This is fallout from renaming storage -> database. I was already making configuration schema changes in this PR, so this felt like a good time to break this.